src/constants.js

```diff
@@ -28,4 +28,5 @@ export const errorTypes = {
   "missing-key": "Key is missing",
   "invalid-header": "Header is not valid",
   "invalid-json": "Json format is not valid",
+  "unnecessary-parameter": "This parameter must not be sent with the current value."
```

Comment:
I'm having a hard time trying to make sense out of this one.
The parameter is not needed or the parameters _must_ not be sent?
I feel like the name of the key does not match the description. If the parameter _must_ not be sent, then the name of the key could probably be different. If it's not needed, then the description could be adjusted to actually describe that is not needed.

Reply:
I'm trying to understand it and I think that some parameters can be sent just with one unique value. 

For example, if a parameter is a boolean that only accepts a 'false' value if you send it in 'true' this is the error to describe it. The way to "send it" in 'true' is not sending it (as it is a default value). 

If this is the case may be an "unnecessary-value" error is more descriptive

Suggestion: "unnecessary-value": "Sending this parameter with the specified value is unnecessary and should not be included in the request."

Reply:
Yes, I now realize it may not be very clear. There are some keys that should not be sent when they are a certain value. For examble, the key 'pr' should only be sent if not equal to 1. This error is meant for this types of situations. I will change the name and description to make it clear. Does this answer your questions?

Reply:
I decided to change the name of the key to ''unnecessary-key'' and the description to 'This key must not be sent with the current value.', which I believe is more understandable. However, any suggestions improve comprehension are more than welcomed.